docker/CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Changelog
 All notable changes to the docker containers will be documented in this file.
 
+### 2021-02-10
+ - Check if messages being scanned by API scan scripts are globally excluded or not.
```

Comment:
```suggestion
 - Check if messages being analyzed by API scan scripts are globally excluded or not.
```

Is that safer/clearer?

Reply:
I know we call "passive" scanning, but I wanted it to stay clear that nothing is sent. I guess "passively scanned" would  be fine too.

Reply:
Changed.